src/BSeries.jl

```diff
@@ -40,6 +40,8 @@ export renormalize!
 
 export is_energy_preserving, energy_preserving_order
 
+export CSRK
```

Comment:
```suggestion
export ContinuousStageRungeKuttaMethod
```
That's how the struct is called below. Currently, we don't use abbreviations, so I would like to keep it like this. Please open an issue for discussing whether we may want to export common abbreviations such as `RK`, `AVF`, or `CSRK` in the future.